CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Removed the deprecated ``CrateClient`` class
```

Comment:
actually we removed the whole client package. pls also add the hint to use the Crate JDBC driver as an alternative